src/components/GroupSystems/index.js

```diff
@@ -4,20 +4,24 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 import NoSystemsEmptyState from '../InventoryGroupDetail/NoSystemsEmptyState';
 import GroupSystems from './GroupSystems';
+import GroupImmutableSystems from './GroupImmutableSystems';
 
-const GroupSystemsWrapper = ({ groupName, groupId }) => {
+const GroupSystemsWrapper = ({ groupName, groupId, hostType }) => {
   const { uninitialized, loading, data } = useSelector(
     (state) => state.groupDetail
   );
-
   return uninitialized || loading ? (
     <EmptyState>
       <EmptyStateBody>
         <Spinner />
       </EmptyStateBody>
     </EmptyState>
   ) : (data?.results?.[0]?.host_count || 0) > 0 ? (
```

Comment:
Can we double-check that data?.results?.[0]?.host_count also includes the immutable systems? If not, then we have to update the condition here too.

Reply:
the results not includes the immutable.
We need explicit pass the filter to get the total of immutable.

Reply:
yes, that is the expected behavior.
If there the total of `conventional is 0 and immutable > 0` we should open the screen with focus in the immutable tab and the conventional will be empty.
![empty](https://github.com/RedHatInsights/insights-inventory-frontend/assets/5039367/3772f3e5-fa2c-43f1-9b8d-d7824b750230)